code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -73,11 +73,9 @@
 	emote_see = list("bubbles", "oozes")
 	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/particle_effect/foam)
 
-/mob/living/simple_animal/hostile/retaliate/clown/lube/Life()
+/mob/living/simple_animal/hostile/retaliate/clown/lube/ComponentInitialize()
 	. = ..()
-	var/turf/open/OT = get_turf(src)
-	if(isopenturf(OT))
-		OT.MakeSlippery(TURF_WET_LUBE, 100)
+	AddElement(/datum/element/snailcrawl) //Not a component

```

Comment:
you dont have to use componentinitialize its a deprecated proc anyways and it would make the comment unnecesary

Reply:
It was supposed to be a joke, turns out it wasn't a good one.

Reply:
i cant tell anymore in this codebase